pkg/security/utils/path_windows.go

```diff
@@ -0,0 +1,13 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package utils
+
+import "strings"
+
+// NormalizePath normalize path
+func NormalizePath(path string) string {
+	return strings.TrimLeft(path, "\\??\\")
```

Comment:
```suggestion
	return strings.TrimPrefix(path, "\\??\\")
```
otherwise TrimLeft works with a cutset so this would give

`NormalizePath("?\whatever") = "whatever"`